lib/hammer_cli/utils.rb

```diff
@@ -48,18 +48,6 @@ def wrap(line_width: 80, break_sequence: "\n")
   end
 end
 
-class Hash
-  # for ruby < 2.5.0
```

Comment:
Does it make sense to set `required_ruby_version` to `>= 2.5` in the gemspec?

Reply:
Well, I'd set it to `>= 2.7` even, since I'm not sure we even test lower versions...